### DISCUSSION TITLE
Question not issue

### DISCUSSION THREAD

**Discussion by PLAN8VR** [domlysz/BlenderGIS#886] [General]
Hi,

Sorry to bother you as I think your wiki already explains it, but Im not getting it to work!

I want to export a georeferenced tiff from Blender, I have set up the geocam and render just fine, but when I import into qgis, the crs doesnt seem to be recognised or correct - can you help me understand how to export the geotiff please? thank you!

**Comment by domlysz** [comment]
You need to save the worldfile which contains georeferencing informations
https://github.com/domlysz/BlenderGIS/wiki/Make-a-georef-render

**Reply by PLAN8VR** [reply]
Hi there,

Thanks for getting back to me (and the recent blender 4.1 update!)

It's been a while since I asked this question, but I think the problem I was having was that I couldn't find the world file on export, but I'll try again when I find a minute!
